config/locales/en.yml

```diff
@@ -470,16 +471,16 @@ en:
         social_homebuy: "Social HomeBuy buyers should not have lived here before"
         rent_to_buy: "Rent to Buy buyers should not have lived here before"
       hodate:
-        must_be_before_saledate: "Practical completion or handover date must be before exchange date"
-        must_be_less_than_3_years_from_saledate: "You told us practical completion or handover date is more than 3 years before completion date"
+        must_be_before_saledate: "Practical completion or handover date must be before sale completion date"
```

Comment:
this one was actually an error that I saw as part of this ticket, rest are just cosmetic changes